frontend/server-side-translations/push-server-strings.js

```diff
@@ -0,0 +1,26 @@
+const { tx } = require('@transifex/native');
```

Comment:
Should we add this to the deployment script? Can you see with @davidsingal?

Reply:
The script `yarn transifex:push` is already on the deploy task